ui_framework/src/components/side_nav/side_nav.js

```diff
@@ -0,0 +1,39 @@
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+import { KuiIcon } from '../../components';
+
+export const KuiSideNav = ({ children, isOpenMobile, className, ...rest }) => {
+  const classes = classNames(
+    'kuiSideNav',
+    className,
+    {
+      'isOpenMobile': isOpenMobile,
+    },
+  );
+
+  return (
+    <nav
+      className={classes}
+      {...rest}
+    >
+      {/* Hidden from view, except in mobile */}
+      <button className="kuiSideNav__mobileToggle kuiLink">
+        <span className="kuiSideNav__mobileTitle">Navigate within management</span>
+        <span><KuiIcon className="kuiSideNav__mobileIcon" type="apps" size="medium"/></span>
```

Comment:
Could you format this and similar JSX like this:

```jsx
<span>
  <KuiIcon
    className="kuiSideNav__mobileIcon"
    type="apps"
    size="medium"
  />
</span>
```

This adheres to our [HTML style guide rules](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md). Oddly, the linter should be catching this. Do you have the latest NPM dependencies installed? You can also run `npm run lintroller` to autofix a lot of this style guide stuff.